setup.py

```diff
@@ -1,18 +1,21 @@
 from os import path
 from setuptools import setup, find_packages
 
+VERSION = '1.0.0'
```

Comment:
```suggestion
VERSION = '0.2.4'
```

Reply:
In that case, I'll probably add some more things to the classifiers section to ensure people know our development status.